### ISSUE TITLE
Add token validation to `PUT` and `DELETE` blog methods - like the one done for `POST`

### ISSUE THREAD

**Issue by tacosontitan** [Vinayaka2k/blog-list-app#32]
Currently, the `POST` method in the blog endpoint requires authentication headers. However, based on the code for the `DELETE` and `PUT` methods, authentication is not required which seems a little counterintuitive when considering that creating a blog does.

Should the `DELETE` and `PUT` methods require authentication headers?

**Comment by tacosontitan** [comment]
If the repository owner confirms that authentication is required, this issue should also serve to track the changes required for implementation.

**Comment by Vinayaka2k** [comment]
yes, I missed that part. PUT and DELETE must require authentication headers as well !

**Comment by tacosontitan** [comment]
I can take this on, feel free to assign me!
